pkg/registry/image_util.go

```diff
@@ -0,0 +1,27 @@
+package registry
+
+import (
+	"github.com/google/go-containerregistry/pkg/v1"
+)
+
+func ImageSize(image v1.Image) (int64, error) {
```

Comment:
```suggestion
func imageSize(image v1.Image) (int64, error) {
```

Could we keep this not exported?

Reply:
the source.Uploader makes use of this. How do you want me to handle this?
previously Source Uploads would happen without any info displayed to the user. I added the Uploading... and Spinner to the source.Uploader

Reply:
marking as resolved. addressed in commit https://github.com/vmware-tanzu/kpack-cli/commit/be211733ecaa33231703f7b9d2938b0b71bee3a3
please reopen if not ok